src/executor/source/ApiInterface.ts

```diff
@@ -0,0 +1,13 @@
+import axios, { AxiosResponse } from 'axios'
+import https from 'https'
+
+const apiBaseAddress = 'https://localhost:8080/api/'
```

Comment:
This address should com from an env variable.

For this you can add the [`dotenv`](https://www.npmjs.com/package/dotenv) package. Then create a `.env` file in the `/src/editor/` folder. 

Now you can add your required env variables to that file and access them in node via `process.env.<NAME>`.

You can also check how it's don on the editor.